src/common/document.js

```diff
@@ -1,12 +1,19 @@
 import { useEffect } from 'react';
+import _ from 'lodash/fp';
 import { useTranslation } from 'react-i18next';
 
-export const useDocumentTitle = (title, fetching) => {
+export const useDocumentTitle = (title, fetching, omitSuffix = false) => {
   const { t } = useTranslation();
 
-  const fullTitle = fetching
-    ? t('common.terraso_projectName')
-    : `${title} | ${t('common.terraso_projectName')}`;
+  const titleParts = [
+    fetching ? null : title,
+    omitSuffix ? null : t('common.terraso_projectName'),
+  ];
+
+  const fullTitle = _.flow(
+    _.filter(part => !_.isEmpty(part)),
+    _.join(' | ')
+  )(titleParts);
```

Comment:
suggestion: simplify this as `_,compact(titleParts).join(' | ')`.

Reply:
Thanks, didn't know about that compact function

Reply:
Any time you are filtering on "not empty" in a verbose way, it's a hint to look for something briefer. :)

In regular JavaScript, you can also do `foo.filter(bar => bar)` and `foo.filter(Number)`.